telemetry/service/service-model.json

```diff
@@ -64,6 +64,21 @@
       "type":"string",
       "max":2000
     },
+    "CommputeEnv":{
+      "type":"string",
+      "enum":[
+        "cloud9",
+        "codeCatalyst",
+        "dev",
```

Comment:
is this for "gamma" or something like that?

Reply:
actually i added two words for dev & local that mean the same. removed it